ocp_utilities/infra.py

```diff
@@ -81,6 +85,55 @@ def assert_pods_failed_or_pending(pods):
         )
 
 
+def assert_nodes_in_healthy_condition(nodes):
```

Comment:
please give the user the option to send the dict to compare from while the full dict will be the default
@dbasunag fyi